src/libs/url_utils.js

```diff
@@ -55,7 +55,7 @@ export function getAppRelativeUrl() {
 }
 
 export function getAppRelativePathname() {
-  const appBase = (window.baseURL || '/').replace(/\/$/, '');
+  const appBase = (window.baseUrl || '/').replace(/\/$/, '');
```

Comment:
Easy mistake, especially as our global `baseUrl` should not be confused with [`document.baseURI`](https://developer.mozilla.org/en-US/docs/Web/API/Node/baseURI) (similar value except it contains a complete URL: protocol, host, etc.)